notification/rocketchat.py

```diff
@@ -0,0 +1,227 @@
+        specifed during the creation of webhook.
+    required: false
+    default: None
+  username:
+    description:
+      - This is the sender of the message.
+    required: false
+    default: "Ansible"
+  icon_url:
+    description:
+      - Url for the message sender's icon (default C(http://www.ansible.com/favicon.ico))
+    required: false
+  icon_emoji:
+    description:
+      - Emoji for the message sender. The representation for the available emojis can be
+        got from Rocket Chat. (for example :thumbsup:) (if I(icon_emoji) is set, I(icon_url) will not be used)
+    required: false
+    default: None
+  link_names:
+    description:
+      - Automatically create links for channels and usernames in I(msg).
+    required: false
+    default: 1
+    choices:
+      - 1
+      - 0
+  validate_certs:
+    description:
+      - If C(no), SSL certificates will not be validated. This should only be used
+        on personally controlled sites using self-signed certificates.
+    required: false
+    default: 'yes'
+    choices:
+      - 'yes'
+      - 'no'
+  color:
+    version_added: "2.1"
+    description:
+      - Allow text to use default colors - use the default of 'normal' to not send a custom color bar at the start of the message
+    required: false
+    default: 'normal'
+    choices:
+      - 'normal'
+      - 'good'
+      - 'warning'
+      - 'danger'
+  attachments:
+    description:
+      - Define a list of attachments.
+    required: false
+    default: None
+"""
+
+EXAMPLES = """
+- name: Send notification message via Rocket Chat
+  local_action:
+    module: rocketchat
+    token: thetoken/generatedby/rocketchat
+    msg: "{{ inventory_hostname }} completed"
+
+- name: Send notification message via Rocket Chat all options
+  local_action:
+    module: rocketchat
+    token: thetoken/generatedby/rocketchat
+    msg: "{{ inventory_hostname }} completed"
+    channel: "#ansible"
+    username: "Ansible on {{ inventory_hostname }}"
+    icon_url: "http://www.example.com/some-image-file.png"
+    link_names: 0
+
+- name: insert a color bar in front of the message for visibility purposes and use the default webhook icon and name configured in rocketchat
+  rocketchat:
+    token: thetoken/generatedby/rocketchat
+    msg: "{{ inventory_hostname }} is alive!"
+    color: good
+    username: ""
+    icon_url: ""
+
+- name: Use the attachments API
+  rocketchat:
+    token: thetoken/generatedby/rocketchat
+    attachments:
+      - text: "Display my system load on host A and B"
+        color: "#ff00dd"
+        title: "System load"
+        fields:
+          - title: "System A"
+            value: "load average: 0,74, 0,66, 0,63"
+            short: "true"
+          - title: "System B"
+            value: "load average: 5,16, 4,64, 2,43"
+            short: "true"
+
+"""
+
+ROCKETCHAT_INCOMING_WEBHOOK = 'https://%s/hooks/%s'
```

Comment:
Might want to make the http or https configurable.  Most people should have https.  But they might not all.